kymatio/scattering3d/frontend/torch_frontend.py

```diff
@@ -39,6 +73,21 @@ def register_filters(self):
         self.register_buffer('tensor_gaussian_filter', self.gaussian_filters)
 
     def scattering(self, input_array):
+        """
```

Comment:
Same here. Shouldn't need a docstring.

Reply:
My bad,  I didn't bother removing this during the cherry-pick